docs/tutorial/kymotracking.rst

```diff
@@ -0,0 +1,157 @@
+
+- Once no more candidates are found in the next `window` frames, the trace is terminated and we proceed by initiating a new trace from the peak which is now the highest.
+
+- Once there are no more peaks in the frame from which we are currently initiating traces, we start initiating traces from the next frame. This process is continued until there are no more peaks left to trace.
+
+The score function is based on a prediction of where we expect future peaks. Based on the peak location of the tip of
+the trace `x` and a velocity `v`, it computes a predicted position over time. The score function assumes a Gaussian
+uncertainty around that prediction, placing the mean of that uncertainty on the predicted extrapolation. The width of
+this uncertainty is given by a base width (provided as sigma) and a growing uncertainty over time given by a diffusion
+rate. This results in the following model for the connection score.
+
+.. math::
+
+    S(x, t) = N\left(x + v t, \sigma_{base} + \sigma_{diffusion} \sqrt{t}\right).
+
+Here `N` refers to a normal distribution. In addition to the model, we also have to set a cutoff, after which we deem
+peaks to be so unlikely to be connected that they shouldn't be. By default, this cutoff is set at two sigma. Scores
+outside this cutoff are set to zero which means they will not be accepted as a new point.
+
+
+Using the greedy algorithm
+--------------------------
+
+So, time to use this algorithm. First, we need to get an image to perform the tracing on. Let's grab a kymograph from
+a Bluelake file::
+
+    file = lk.File('kymograph.h5')
+    _, kymo = file.kymos.popitem()
+
+We can extract the image data with one of the `_image` commands::
+
+    data = kymo.green_image
+
+Let's have a look at what it looks like::
+
+    plt.imshow(data)
+
+It seems that our traces in this image are about 5 pixels wide, so let's set this as our `line_width`. The peaks have an
+intensity of about 5 photon counts, whereas the background fluctuates around 2-3. Let's set our `pixel_threshold` to 4.
+We also see that sometimes, a particle momentarily disappears. To still connect these in a single trace, we want to
+allow for some gaps in the connection step. Let's use a `window` size of 4 in this test. Running the algorithm is easy::
+
+    traces = track_particles_greedy(data, line_width=5, pixel_threshold=4, window=4)
+
+The result is a set of kymograph traces. We can get the determined time and coordinates of these traces from the
+`time` and `coordinate` attributes. Let's plot the first trace::
+
+    plt.plot(traces[0].time, traces[0].coordinate)
+
+Plotting all the lines is also quite easy using a Python list comprehension::
+
+    [plt.plot(trace.time, trace.coordinate for trace in traces]
+
+Note that the determined time and coordinates of the traces and coordinates are defined in pixels and therefore need to
+be converted to actual time and position using the delta time and pixel size of the kymograph. We can get the time
```

Comment:
I missed this detail during my review of the previous PR. Could we maybe rename the properties `.time` and `.coordiante` to `.time_idx` and `.coordinate_idx` to better reflect this? I think this is an easy mistake otherwise.

Reply:
Yeah, I considered that as well already but was on the fence about it. But will do.